### PR TITLE
[docs] ldap platform docs

### PR DESCRIPTION
**Pull request by kerneljake**
LDAP instructions for Yugabyte Platform.

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/security/ldap-authentication-platform.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+title: LDAP Authentication
+headerTitle: LDAP Authentication
+linkTitle: LDAP Authentication
+description: Configuring Yugabyte Platform to use an external LDAP authentication service.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/security/ldap-authentication-platform.md:5]: Titles should all be 'LDAP authentication'

File: docs/content/latest/secure/authentication/ldap-authentication.md [hunk 2]
```diff
@@ -214,19 +214,19 @@ In the above sample configuration, we are using an [online LDAP test server](htt
...
-1. Configure database role(s) for the LDAP user(s)
+1. Configure database role(s) for the LDAP user(s).
 
-    ```sql
-    We are creating a ROLE for username riemann supported by the test LDAP server. 
+    We are creating a `ROLE` for username riemann supported by the test LDAP server. 
 
+    ```sql
     yugabyte=# CREATE ROLE riemann WITH LOGIN;
     yugabyte=# GRANT ALL ON DATABASE yugabyte T0 riemann;
```
Comment (polarweasel) [inline docs/content/latest/secure/authentication/ldap-authentication.md:226]: Not sure T0 (T zero) is correct here

File: docs/content/latest/yugabyte-platform/security/ldap-authentication-platform.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+    ```output
+    Password for user yugabyte: yugabyte
+    ysqlsh (11.2-YB-2.5.1.0-b0)
+    Type "help" for help.
+    ```
+
+    ```sql
+    exampledb=# create user adam;
+    CREATE ROLE
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-platform/security/ldap-authentication-platform.md:117]: Don't need this line

**Review by polarweasel — COMMENTED** [review]
Double check the T0 line. Other comments can be ignored if you want. :)

**Pull request merged**